Sources/String+Polymorphic.swift

```diff
@@ -85,6 +85,13 @@ extension String: Polymorphic {
     public var object: [String : Polymorphic]? {
         return nil
     }
+
+    /**
+        Converts the string to a UTF8 array of bytes.
+    */
+    public var bytes: [UInt8]? {
+        return [UInt8](self.utf8)
```

Comment:
Is `String.utf8` the same thing as `string.bytes` that we have in Core? Might be worth importing Core here for that method. It at least used to be the only reliable method on both macOS and Linux.